File: library/src/main/java/cn/wandersnail/widget/recyclerview/OnStartDragListener.java
package cn.wandersnail.widget.recyclerview;

import androidx.recyclerview.widget.RecyclerView;

/**
 * 拖拽监听器
 * <p>
 * date: 2019/8/22 19:20
 * author: zengfansheng
 */
public interface OnStartDragListener {
    /**
     * 当开始拖拽时
     *
     * @param viewHolder 被拖拽的view的持有者
     */
    void onStartDrag(RecyclerView.ViewHolder viewHolder);
}

File: library/src/main/java/com/snail/widget/helper/OnStartDragListener.kt
package com.snail.widget.helper

import androidx.recyclerview.widget.RecyclerView

/**
 *
 *
 * date: 2019/3/9 22:44
 * author: zengfansheng
 */
interface OnStartDragListener {
    /**
     * 当开始拖拽时
     * 
     * @param viewHolder 被拖拽的view的持有者
     */
    fun onStartDrag(viewHolder: RecyclerView.ViewHolder)
}